src/applications/disability-benefits/2346/schemas/definitions/2346UI.js

```diff
@@ -22,48 +27,71 @@ const emailUIDescription = (
   </>
 );
 
+const selectedAddressUITitle = (
+  <h4 className="vads-u-display--inline">Shipping address</h4>
+);
+
+const selectedAddressUIDescription = (
+  <>
+    <br />
```

Comment:
we need to avoid using breaks and use the utility classes to style appropriately